lib/hydra/pcdm/services/collection/remove_related_object.rb

```diff
@@ -0,0 +1,33 @@
+module Hydra::PCDM
+  class RemoveRelatedObjectFromCollection
+
+    ##
+    # Remove an object from a collection.
+    #
+    # @param [Hydra::PCDM::Collection] :parent_collection from which to remove the related object
+    # @param [Hydra::PCDM::Object] :child_related_object being removed
+    #
+    # @return [Hydra::PCDM::Collection] the updated pcdm collection
+
+    def self.call( parent_collection, child_related_object )
+      raise ArgumentError, "parent_collection must be a pcdm collection" unless Hydra::PCDM.collection? parent_collection
+      raise ArgumentError, "child_related_object must be a pcdm object" unless Hydra::PCDM.object? child_related_object
+
+
+      # TODO FIX when related_objects is empty, related_objects.delete raises ActiveFedora::ObjectNotFoundError "Can't reload an object that hasn't been saved"  (activefedora-aggregation issue #35)
```

Comment:
projecthydra-labs/activefedora-aggregation#35 is about members, not related-objects, right?